save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/organizations/OrganizationToolsMenu.kt

```diff
@@ -57,12 +57,27 @@ external interface OrganizationToolsMenuProps : Props {
     /**
      * lambda for update projects
      */
-    var updateProjects: (MutableList<ProjectDto>) -> Unit
+    var updateProjects: (List<ProjectDto>) -> Unit
 }
 
 @Suppress("TOO_LONG_FUNCTION", "LongMethod", "CyclomaticComplexMethod")
 private fun organizationToolsMenu() = FC<OrganizationToolsMenuProps> { props ->
     val (projects, setProjects) = useState(props.projects)
+    val comparator: Comparator<ProjectDto> =
+            compareBy<ProjectDto> { orderedProjectStatus[it.status] }
+                .thenBy { it.name }
+
+    /**
+     * @param project
+     * @param newStatus
+     */
+    @Suppress("AVOID_NESTED_FUNCTIONS")
+    fun localButtonAction(project: ProjectDto, newStatus: ProjectStatus) {
```

Comment:
It either should be private or in another file, shouldn't it?

Reply:
this function is described inside another private function

Reply:
And why so? It doesn't seem to be a good practice

Reply:
Took out a function for another function